Makefile.am

```diff
@@ -148,7 +148,7 @@ EXTRA_DIST = \
     lib/sapi.pc.in \
     man/man-postlude.troff \
     man/InitDeviceTcti.3.in \
-    man/man3/InitSocketTcti.3 \
```

Comment:
I don't think this belong to this commit, right? Instead you should cherry-pick from 1.x branch commit 784695d69c6e ("build: Distribute the source for the InitSocketTcti man page.").

Reply:
Yes, that's some leftovers from Philip's PR that somehow got into my branch. Let me rebase. Thanks for catching that @martinezjavier